src/main/java/com/espro/flink/consul/jobgraph/ConsulSubmittedJobGraphStore.java

```diff
@@ -78,4 +79,10 @@ public void removeJobGraph(JobID jobId) throws Exception {
 	private String path(JobID jobID) {
 		return jobgraphsPath + jobID.toString();
 	}
+
+    @Override
+    public void releaseJobGraph(JobID jobId) throws Exception {
+        // TODO Auto-generated method stub
```

Comment:
Is the implementation missing?

Reply:
Good Question! I will check it, if i can keep it with an empty implementation and therefore simply remove the autogenerated comment. Sry.

Reply:
I checked the method. There is no need for proper implementing this, only added a comment.